src/layout/OrganisationLookup/OrganisationLookupComponent.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useState } from 'react';
+
+import { ErrorMessage } from '@digdir/designsystemet-react';
+import { queryOptions, useQuery } from '@tanstack/react-query';
+
+import type { PropsFromGenericComponent } from '..';
+
+import { Button } from 'src/app-components/Button/Button';
+import { NumericInput } from 'src/app-components/Input/NumericInput';
+import { Fieldset } from 'src/app-components/Label/Fieldset';
+import { Label } from 'src/app-components/Label/Label';
+import { Description } from 'src/components/form/Description';
+import { RequiredIndicator } from 'src/components/form/RequiredIndicator';
+import { getDescriptionId } from 'src/components/label/Label';
+import { useDataModelBindings } from 'src/features/formData/useDataModelBindings';
+import { Lang } from 'src/features/language/Lang';
+import { useLanguage } from 'src/features/language/useLanguage';
+import { ComponentValidations } from 'src/features/validation/ComponentValidations';
+import { useBindingValidationsForNode } from 'src/features/validation/selectors/bindingValidationsForNode';
+import { hasValidationErrors } from 'src/features/validation/utils';
+import { ComponentStructureWrapper } from 'src/layout/ComponentStructureWrapper';
+import classes from 'src/layout/OrganisationLookup/OrganisationLookupComponent.module.css';
+import { validateOrganisationLookupResponse, validateOrgnr } from 'src/layout/OrganisationLookup/validation';
+import { useLabel } from 'src/utils/layout/useLabel';
+import { useNodeItem } from 'src/utils/layout/useNodeItem';
+import { httpGet } from 'src/utils/network/networking';
+import { appPath } from 'src/utils/urls/appUrlHelper';
+
+const orgLookupQueries = {
+  lookup: (orgNr: string) =>
+    queryOptions({
+      queryKey: [{ scope: 'organisationLookup', orgNr }],
+      queryFn: () => fetchOrg(orgNr),
+      enabled: false,
+      gcTime: 0,
+    }),
+};
+
+export type Organisation = {
+  orgNr: string;
+  name: string;
+};
+export type OrganisationLookupResponse =
+  | { success: false; organisationDetails: null }
+  | { success: true; organisationDetails: Organisation };
+
+async function fetchOrg(orgNr: string): Promise<{ org: Organisation; error: null } | { org: null; error: string }> {
+  if (!orgNr) {
+    throw new Error('orgNr is required');
+  }
+  const url = `${appPath}/api/v1/lookup/organisation/${orgNr}`;
+
+  try {
+    const response = await httpGet<{ OrganisationLookupResponse: OrganisationLookupResponse }>(url);
```

Comment:
Tror ikke du trenger å passe type her siden du validerer med ajv nedenfor og dermer får typen (og er sikker på at dataen _faktisk_ ser sånn ut) 👌 

```suggestion
    const response = await httpGet(url);
```